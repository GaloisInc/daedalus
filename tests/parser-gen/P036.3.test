P036.ddl
--gen
--interp=inputs/P036.3.inp
