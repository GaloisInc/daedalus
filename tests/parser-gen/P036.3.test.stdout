[WARNING] ./P036.ddl:4:9--4:35:
            Using unbiased choice may be inefficient.
--- Found 1 results:
{ x: '3'
, y: {c3 : { }}
}
